src/charactersheet/viewmodels/dm/encounter_sections/point_of_interest_section/index.html

```diff
@@ -141,33 +142,6 @@
               </span>
             </div>
           </div>
-          <div class="form-group ui-front">
-            <label class="col-sm-2 control-label">
-              Required DC
```

Comment:
why is the dc stuff being removed?

Reply:
I think this was a merge issue. I added it back in my other PR.